be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -270,9 +270,10 @@ Status ExchangeSinkOperator::Channel::send_one_chunk(RuntimeState* state, const
     return Status::OK();
 }
 
-Status ExchangeSinkOperator::Channel::send_chunk_request(RuntimeState* state, PTransmitChunkParamsPtr chunk_request,
-                                                         const butil::IOBuf& attachment,
-                                                         int64_t attachment_physical_bytes) {
+[[nodiscard]] Status ExchangeSinkOperator::Channel::send_chunk_request(RuntimeState* state,
```

Comment:
this nodiscard can be removed, adding it where the function is declared is enough